modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/authenticator/management/v1/AuthenticatorSuccessTest.java

```diff
@@ -57,6 +57,10 @@ public class AuthenticatorSuccessTest extends AuthenticatorTestBase {
     private UserDefinedLocalAuthenticatorCreation creationPayload;
     private UserDefinedLocalAuthenticatorUpdate updatePayload;
 
+    private final String CUSTOM_TAG = "Custom";
+    private final String[] CURRENT_TAGS_LIST = new String[]{"APIAuth","MFA","Passwordless","Passkey",
+            "Username-Password", "Request-Path","Social-Login","OIDC","SAML","Enterprise"};
```

Comment:
```suggestion
    private final String[] TAGS_LIST = new String[]{"APIAuth","MFA","Passwordless","Passkey",
            "Username-Password", "Request-Path","Social-Login","OIDC","SAML","Enterprise","Custom"};
```

Shall we use this as the general tag list and add "Custom" tag also ? It can ensure the reusability of the variable with future modifications if necessaray.

Reply:
https://github.com/wso2/product-is/pull/22059#discussion_r1889667131